vector-config/src/main/java/im/vector/app/config/Config.kt

```diff
@@ -43,7 +43,7 @@ object Config {
     /**
      * The maximum length of voice messages in milliseconds.
      */
-    const val VOICE_MESSAGE_LIMIT_MS = 120_000L
+    const val VOICE_MESSAGE_LIMIT_MS = 900_000L
```

Comment:
For better readability, we could use this nice Kotlin syntax: `val VOICE_MESSAGE_LIMIT_MS = 15.minutes.inWholeMilliseconds`

Reply:
Oh nice, I wasn't aware of this. Made the change.